README.md

```diff
@@ -39,75 +40,30 @@ Roasted Tomahtoes is a sensible, aesthetically engaging, and user friendly movie
 
 ## Gif Showcase
 
-<!-- ![gif of viewing manager dashboard](./src/images/overlook-mgr-dash.gif)</br>
-*Viewing manager login and dashboard, including daily metrics and search functionality*
-
-![gif of manager view all guests info](./src/images/overlook-mgr-view-guests.gif)</br>
-*Viewing manager dashboard, reviewing guest information*
+[gif of homeview, incorrect login, correct login](./src/images/home-login.gif)</br>
+*Viewing guest home view, incorrect & correct login*
 
-![gif of logging in and viewing as guest](./src/images/overlook-guestview.gif)</br>
-*Logging in and viewing as guest*
+[gif of single movie page](./src/images/showMovie.gif)</br>
+*Viewing single movie page with trailers, info, and comment section*
 
-![gif of filtering rooms](./src/images/overlook-filter-rms.gif)</br>
-*Default date, cost, and all rooms returned, unless otherwise selected*
+[gif of search functionality](./src/images/search.gif)</br>
+*Searching various movies*
 
-![gif of searching, adding, deleting reservations](./src/images/overlook-add-delete-reserv.gif)</br>
-*Searching, adding, and deleting reservations*
+[gif of favoring movies](./src/images/favoringMovies.gif)</br>
+*Display user's movies, favoriting and unfavoriting movies*
 
-![gif of error message search results not found](./src/images/overlook-error.gif)</br>
-*Viewing error message when rooms are unavailable*
+[gif of sorting by genre and loading gif](./src/images/sortByGenre.gif)</br>
+*Sort by genre and show loading gif*
 
-![gif of viewing alert when username is incorrect](./src/images/overlook-wronglogin.gif)</br>
-*Alert when username is incorrect*
+[gif of commenting on movies](./src/images/commenting.gif)</br>
+*Viewing comments added to movie*
 
-![gif of accessibility score](./src/images/overlook-accessibility.gif)</br>
-*Tabbing through elements on page, 100% Lighthouse score*
-
-![gif of mobile view login](./src/images/overlook-mobile.gif)</br>
-*Mobile view* -->
+[gif of different views on guest and user login](./src/images/guestView.gif)</br>
+*When logged in, user can rate and comment, but cannot as a guest*
 
 ## Reflections
 
+This project introduced our paired team to React.js and Router. We feel we have demonstrated our understanding of these technologies by satisfying all demands of the spec, as well as completing all extensions, and even adding our own features, all while maintaining a 97% Accessibility score. We implemented TDD using Jest to test for UX and async elements. If we had more time, we would have styled the app mobile-friendly, fixed a bug that caused our page to load the home page when the refresh button is clicked, added more tests, and displayed all comments about movies from the community (not just the user).  Our project documentation can be viewed on [Project Board](https://github.com/GreysonElkins/roasted-tomahtoes/projects) and in the aforementioned gif section. We worked well together and really crushed this introductory React project. 
```

Comment:
What features did we add? 

Maybe add "added alt-text" to emojis to "If we had more time... " ?